doc/source/visualization.rst

```diff
@@ -2,25 +2,31 @@
 .. _visualization:
 
 .. ipython:: python
+   :flake8-add-ignore: E702, E703
```

Comment:
not sure why this is ignored, I'd prefer to not ignore much, but if needed please add a comment on what is being ignored